python_orocos_kdl_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(python_orocos_kdl_vendor)
+
+find_package(ament_cmake REQUIRED)
+
+option(FORCE_BUILD_VENDOR_PKG
+  "Build python_orocos_kdl from source, even if system-installed package is available"
+  OFF)
+
+if(NOT FORCE_BUILD_VENDOR_PKG)
+  # Check if Python bindings are installed by trying to import from interpreter
+  # Figure out Python3 debug/release before anything else can find_package it
+  if(WIN32 AND CMAKE_BUILD_TYPE STREQUAL "Debug")
+    find_package(python_cmake_module REQUIRED)
+    find_package(PythonExtra REQUIRED)
+
+    # Force FindPython3 to use the debug interpreter where ROS 2 expects it
+    set(Python3_EXECUTABLE "${PYTHON_EXECUTABLE_DEBUG}")
+  endif()
+  find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+  execute_process(
+    COMMAND ${Python3_EXECUTABLE} -c "import PyKDL"
+    RESULT_VARIABLE pykdl_import_exit_code
+    OUTPUT_QUIET
+    ERROR_QUIET
+  )
+  if(pykdl_import_exit_code EQUAL 0)
+    set(pykdl_FOUND 1)
+  endif()
+endif()
+
+macro(build_pykdl)
+  include(FetchContent)
+
+  # This version must match orocos_kdl_vendor exactly
+  fetchcontent_declare(
+    python_orocos_kdl
+    URL https://github.com/orocos/orocos_kinematics_dynamics/archive/11d8bd62128092ff64ab20e419b295ca72aada32.zip
+    URL_MD5 0569a66c811ac80b03febf5f06a21650
+  )
+
+  fetchcontent_getproperties(python_orocos_kdl)
+  # TODO(sloretz) use FetchContent_MakeAvailable() when CMake 3.14 is the minimum
```

Comment:
This can be completely ignored, but I do think CMake 3.14 can be made the minimum here.  This package will only ever be used for Humble and forwards, and looking at https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027 , all of the platforms have CMake >= 3.14.

Reply:
I wasn't sure about the `3.16.1 / 3.12.2` mention in the OpenEmbedded column. Does that mean there's a version of OpenEmbedded where the CMake version is 3.12.2?

Reply:
I guess we'll hold off on changing this unless REP-2000 is updated.

Reply:
Yeah, that's fine.  It's a really minor point, I'm fine with the workaround we have in place.